treeherder/perf/fixtures/performance_bug_templates.yaml

```diff
@@ -7,7 +7,7 @@
     default_product: Firefox
     cc_list: jmaher@mozilla.com, rwood@mozilla.com, ionut.goldan@softvision.ro
     text: |
-      Talos has detected a Firefox performance regression from push {{ revision }}. As author of one of the patches included in that push, we need your help to address this regression.
+      Talos has detected a Firefox performance regression from push {{ revisionHref }}. As author of one of the patches included in that push, we need your help to address this regression.
```

Comment:
Could you change this to (here and below):

```
Talos has detected a Firefox performance regression from push:

 {{ revisionHref }}

As author of one of the...
```

I think it reads a bit better if the link is on a single line.